README.md

```diff
@@ -35,15 +35,45 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ## A to Z Order of Companies
 
-### A
+### A  
+
+### C  
+#### **Camunda**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**: 4 pull requests to any Camunda repo
```

Comment:
Can you please make this a hyperlink to their repos?

Reply:
They have multiple repositories that one can contribute to, so i have only added the main one.